packages/gestalt/src/Borders.css

```diff
@@ -30,7 +30,7 @@
 }
 
 .borderColorLightGray {
-  border-color: var(--gestalt-border-color-lightGray);
+  border-color: var(--gestalt-colorGray100);
```

Comment:
@ponori Are you ok with us changing most of the borders from `#ddd` to `#efefef` in light mode?

<img width="1078" alt="Screen Shot 2020-07-09 at 8 14 51 AM" src="https://user-images.githubusercontent.com/1767822/87061073-26f9ad00-c1c0-11ea-8140-e1aff6bb60cf.png">

I think we need a new color variable for the borders we can override in dark mode

Reply:
Resolved in  https://github.com/pinterest/gestalt/pull/1034